package-testing/php-sdk-test/.env.EXAMPLE

```diff
@@ -0,0 +1,7 @@
+# Set your favourite settings and save this file as .env
```

Comment:
note: planning to change the root directory here to something like `php-sdk-relay` to mirror the docker image name. Open to suggestions!